Backend/tests/test_user_ctrl.py

```diff
@@ -88,19 +92,17 @@ def setUpClass(self):
   def tearDownClass(self):
     table = db.getTable('users', dynamodb).delete()
 
-  '''
-  WIP - will clean up later
   def test_isEmailInSystem(self):
     print("Running isEmailInSystem Test")
 
     email = "test@mentii.com"
     response = usr.isEmailInSystem(email, dynamodb)
-    self.assertTrue(response)'''
+    self.assertTrue(response)
 
-  def test_isEmailInSystem_fail(self):
-    print("Running isEmailInSystem fail case Test")
+  def test_isEmailNotInSystem(self):
+    print("Running isEmailNotInSystem Test")
 
-    email = "no_test@mentii.com"
+    email = "notInDB@mentii.com"
```

Comment:
Minor but we are mentii.me not mentii.com so if notInDB was a real user at mentii.com they would be getting that email. Might be worth changing.